### ISSUE TITLE
Buttons/Switches make sounds when activated by logic

### ISSUE THREAD

**Issue by Midge57** [tsunamayo/Starship-EVO#3281]
### Short Description of the issue:

Buttons/Switches activated by logic produce sound.

### Steps to reproduce the issue:

Activate a switch or button with logic/timers.

### Environment
- Starship EVO version: 20w48a
- Graphics Card: 1060 6GB

**Comment by tsunamayo** [comment]
Hi, this is the expected behaviour... Do you think it is weird? Thanks

**Comment by Midge57** [comment]
Ahh, ProPeach said it wasn't supposed to happen and got my hopes up! Thanks Tsuna.

**Comment by ProPeach** [comment]
I was sure they used to be muted when activated by logic back in the day, otherwise they're super loud for clocks!

**Comment by Midge57** [comment]
Yeah the buttons are extremely loud at the moment. Logic activated buttons not making noises would be amazing.
